docs/package.json

```diff
@@ -73,8 +65,8 @@
     "ember-resolver": "^13.1.0",
     "ember-source": "~6.0.1",
```

Comment:
This is where v6 is coming from.
I forgot you had another app in here
Leaving this at v6 is incompatible with the public hoist pattern, so i think that may need to go, or the ci would need to also install ember-source for this app as well during each scenario